inference-engine/src/transformations/src/transformations/op_conversions/gather_normalize_negative_indices.cpp

```diff
@@ -63,11 +63,9 @@ ngraph::pass::GatherNegativeConstIndicesNormalize::GatherNegativeConstIndicesNor
             ngraph::opset7::Constant::create(input_type, Shape{}, {axis_value}), ngraph::opset7::Constant::create(input_type, Shape{}, {0}));
 
         auto add = std::make_shared<ngraph::opset7::Add>(input_gather, indices_constant);
-        auto gather_new = gather_node->copy_with_new_inputs({data, add, axis_constant});
-        gather_new->set_friendly_name(gather->get_friendly_name());
+        gather->input(1).replace_source_output(add);
```

Comment:
I think in most cases this Add will be folded into constant. So please use `get_constant_from_source` here to avoid CF after.

Reply:
Done.